sql/files/defaultdata/hs/run

```diff
@@ -10,6 +10,9 @@ MAINSOURCE="$1"
 # Set non-existing HOME variable to make GHC program happy, see:
 # https://ghc.haskell.org/trac/ghc/ticket/11678
 export HOME=/does/not/exist
+# Allow temporary files during compilation, this directory is not
+# available during the submission run.
+export TMPDIR=`pwd`
```

Comment:
is `$PWD` defined?

Reply:
`pwd` is the command to get the current work directory. Why do you prefer `$PWD` over `pwd`?

Reply:
`pwd` spawns a process, `$PWD` just reads an environment variable